include/realtime_tools/realtime_publisher.h

```diff
@@ -213,14 +196,12 @@ class RealtimePublisher
 
       // Sends the outgoing message
       if (keep_running_)
-        publisher_.publish(outgoing);
+        publisher_->publish(outgoing);
     }
     is_running_ = false;
   }
 
-  std::string topic_;
-  ros::NodeHandle node_;
-  ros::Publisher publisher_;
+  typename rclcpp::Publisher<Msg>::SharedPtr publisher_;
```

Comment:
A question of style: I think it'd be simpler and more maintainable to add a private/protected `using PublisherSharedptr rclcpp::Publisher<Msg>::SharedPtr;` at the top, then use the alias from then on